extensions/amp-analytics/0.1/vendors/retargetly.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const RETARGETLY_CONFIG = /** @type {!JsonObject} */ ({
+  'requests': {
+    'host': 'https://api.retargetly.com',
+    'page': '${host}/api?id=${accountId}&src=${sourceId}&url=${sourceUrl}' +
+    '&n=${title}&ref=${documentReferrer}&ua=${userAgent}' +
+    '&random=${random}&bl=${browserLanguage}&source=amp',
+  },
+  'transport': {
+    'beacon': false,
```

Comment:
Just to be sure, your server does NOT support xhr or beacon requests? We have a fallback in place for image requests, so that you do not need to enforce only image requests. We can handle dynamically sending the correct request per browser.

If you server supports beacon and/or xhr these should be enabled 😄

Reply:
Hi there torch!

Actually our servers can handle XHR requests but not for this particular endpoint.

In this case we only expect to return an image or a JavaScript code, but not to return just a 200 status without returning a body. We could add this in the future and make another pull request with the changes, do you think that could work?

Reply:
Awesome!

Makes sense, yes, if you add the functionality to your server, feel free to update this and mention me in the PR and we can get that changed 😄

Reply:
Thanks !